### PR TITLE
Frontend: create account page

### LINKED ISSUE #197: Frontend: Create account page
## User story

**Who**: As a developer
**What**: I want to develop the create account page
**Why**: So that I can display the products that have special offers.

## Acceptance criteria
This page should look according to screenshot below:

image

## Implementation details

You will need to include in the Create Account Page container the components and elements for:
Create New Account component.

Endpoint for create new account.

Use the container and style file mentioned on this [issue#32](https://github.com/HackYourFuture-CPH/fp-class19/issues/32)

### PR DESCRIPTION
**Pull request by FlorentinaPetica**
# Description

This pr is for CreateAccountContainer

Fixes #197

# How to test?

Run : npm run dev
got to /sign-up

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by Impressiv3** [comment]
Looks really nice!
The only suggestion that I have is to change the mobile input field type to text.
( what I have in mind is that sometimes people want to give their phone numbers like: +45 26578555 )
image

**Review by Impressiv3 — APPROVED** [review]
Looks really nice. Well done.

**Review by saloumeh-67 — APPROVED** [review]
it works:) .

File: src/client/components/CreateAccountForm/CreateAccountForm.component.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            name="country"
+            value={country}
+            onChange={(event) => setCountry(event.target.value)}
+            required
+          />
+        </label>
+        <label htmlFor="zipcode">
+          Zip-code
+          <input
+            type="text"
```
Comment (Divyajg) [inline src/client/components/CreateAccountForm/CreateAccountForm.component.js:149]: zipcode can be type="number" .

**Review by Divyajg — APPROVED** [review]
Looks good

**Pull request merged**